crypto/src/mls/client/id.rs

```diff
@@ -16,6 +16,8 @@
 
 use crate::CryptoError;
 
+/// A Client identifier
+///
```

Comment:
Is this intentional? It seems incomplete

Reply:
clippy was complaining about paragraphs being too long... I just added something resembling a title